CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog]
 
 ## [Unreleased]
 
+## [Release 012] - 2019-09-30
```

Comment:
I've dated this for Monday, since we shouldn't really do this at this time on a Thursday.